OpenLIFUData/OpenLIFUData.py

```diff
@@ -466,13 +547,69 @@ def onAddNewSubjectClicked(self, checked:bool) -> None:
                 #Update loaded subjects view
                 self.updateSubjectSessionSelector()
     
+    @display_errors
+    def getSelectedSubjectSession(self, index: qt.QModelIndex) -> Tuple[str, str]:
+        """ Returns the subject or session (name, id) currently selected in the SubjectSessionView """
+        
+        name = self.subjectSessionItemModel.itemFromIndex(index.siblingAtColumn(0)).text()
+        id = self.subjectSessionItemModel.itemFromIndex(index.siblingAtColumn(1)).text()
+        
+        return (name, id)
```

Comment:
Just a comment about the function name and docstring -- it's not necessarily the _currently selected_ info that is returned, but rather the info inside the given `index`, right? If I understood that correctly maybe we can remove "selected" from the function name in some way and adjust the docstring

Reply:
You're right. I had initially structured the function differently such that it was retrieving the current index but then changed things around. I'll update the name/doc strings to match the current implementation.